tests/gradients/parameter_shift/test_parameter_shift.py

```diff
@@ -2570,7 +2571,7 @@ def test_output_shape_matches_qnode_probs(self, cost, exp_shape, exp_type):
         assert np.allclose(np.squeeze(np.array(res_parshift)).shape, exp_shape)
 
     # TODO: revisit the following test when the Autograd interface supports
-    # parameter-shift with the new return type system
+    #       parameter-shift with the new return type system
```

Comment:
What in particular are we needing to revist? Given we fully ported to the new return type system two years ago

Reply:
We may also want to rewrite this test to use the new device interface at some point.

Reply:
> What in particular are we needing to revisit?

I don't know 😅 but yeah we definitely want to rewrite many of the gradient tests in general at some point.